core/src/main/java/com/scalar/db/api/AbacAdmin.java

```diff
@@ -0,0 +1,757 @@
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Adds a group to a user with the given username for the given policy. Before adding the group,
+   * levels must be set to the user.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param groupShortName the short name of the group to set
+   * @param accessMode the access mode
+   * @param defaultGroup whether the group is the default group
+   * @param rowGroup whether the group is the row group
+   * @throws ExecutionException if the operation fails
+   */
+  default void addGroupToUser(
+      String policyName,
+      String username,
+      String groupShortName,
+      AccessMode accessMode,
+      boolean defaultGroup,
+      boolean rowGroup)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Removes a group from a user with the given username for the given policy.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param groupShortName the short name of the group to remove
+   * @throws ExecutionException if the operation fails
+   */
+  default void removeGroupFromUser(String policyName, String username, String groupShortName)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Drops the user access for the given username for the given policy.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @throws ExecutionException if the operation fails
+   */
+  default void dropUserAccess(String policyName, String username) throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Retrieves the user tag information of a user with the given username for the given policy.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @return the user tag information. If the user tag information is not registered, returns an
+   *     empty optional
+   * @throws ExecutionException if the operation fails
+   */
+  default Optional<UserTagInfo> getUserTagInfo(String policyName, String username)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Applies a policy to a namespace.
+   *
+   * @param policyName the policy name
+   * @param namespaceName the namespace name
+   * @throws ExecutionException if the operation fails
+   */
+  default void applyNamespacePolicy(String policyName, String namespaceName)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Enables a namespace policy.
```

Comment:
Not sure, but is it more proper?
```suggestion
   * Enables the given policy for the given namespace.
```